tests/regrtest_data/hang/pkg4972/flask/__init__.py

```diff
@@ -0,0 +1,7 @@
+import flask
```

Comment:
Is it necessary to do this with `flask`? It is not currently part of our test dependencies. I haven't tested this and perhaps this works even without the dependency installed but I would prefer a more general case.

Reply:
We can test on "flask looking code" as long as we don't really have to install flask for test ;)

Reply:
Wouldn't it still be better to use "pylintic" imports?

Reply:
I took a few shots at trying to get it working with just standard library imports but eventually gave up. I'll give it another shot.

Reply:
I got a solution working with `builtins.string`.

> Wouldn't it still be better to use "pylintic" imports?

I'm unclear on what you mean by "pylintic" imports sorry.

Reply:
Nice work!

Yeah, that's a bit vague. I meant something like "modules/libraries pylint contributors are familiar with", so something within the `pylint` framework or standard lib. Using `string` is a perfect solution imo!